libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/SlackFile.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Adapters.Slack
+{
+    public class SlackFile
+    {
+        [JsonProperty(PropertyName = "id")]
+        public string Id { get; set; }
+
+        [JsonProperty(PropertyName = "thumb_480")]
+
+        public string Thumb480 { get; set; }
+        
+        [JsonProperty(PropertyName = "idthumb_480_w")]
+        public int Thumb480W { get; set; }
+        
+        [JsonProperty(PropertyName = "thumb_480_h")]
+        public int Thumb480H { get; set; }
+        
+        [JsonProperty(PropertyName = "permalink")]
+        public string Permalink { get; set; }
+        
+        [JsonProperty(PropertyName = "permalink_public")]
+        public string PermalinkPublic { get; set; }
+        
+        [JsonProperty(PropertyName = "edit_link")]
+        public string EditLink { get; set; }
+        
+        [JsonProperty(PropertyName = "preview")]
+        public string Preview { get; set; }
+        
+        [JsonProperty(PropertyName = "preview_highlight")]
+        public string PreviewHighlight { get; set; }
+        
+        [JsonProperty(PropertyName = "lines")]
+        public int Lines { get; set; }
+        
+        [JsonProperty(PropertyName = "lines_more")]
+        public int LinesMore { get; set; }
+        
+        [JsonProperty(PropertyName = "is_public")]
+        public bool IsPublic { get; set; }
+
+        [JsonProperty(PropertyName = "public_url_shared")]
+        public bool PublicUrlShared { get; set; }
+
+        [JsonProperty(PropertyName = "display_as_bot")]
+        public bool DisplayAsBot { get; set; }
+
+        [JsonProperty(PropertyName = "channels")]
+        public string[] Channels { get; set; }
```

Comment:
This is probably where the build is failing. Need to supress the "Properties should not have arrays" on this...